packages/nodes-base/nodes/Strava/ActivityDescription.ts

```diff
@@ -48,6 +42,12 @@ export const activityOperations: INodeProperties[] = [
 				description: 'Get all activity laps',
 				action: 'Get all activity laps',
 			},
+			{
+				name: 'Get Many',
+				value: 'getAll',
+				description: 'Get all activities',
+				action: 'Get all activities',
+			},
```

Comment:
here as well

Reply:
As discussed, order should take into account the new name, so this is correct.